Objects/moduleobject.c

```diff
@@ -698,6 +698,24 @@ module_repr(PyModuleObject *m)
     return PyObject_CallMethod(interp->importlib, "_module_repr", "O", m);
 }
 
+int
+_PyModuleSpec_IsInitializing(PyObject *spec)
+{
+    if (spec != NULL && spec != Py_None) {
```

Comment:
Maybe just return 0 (without calling PyErr_Clear) if this condition is false? It would avoid one identation level :-)

Reply:
We need to call `PyErr_Clear()` in all cases except `spec == Py_None`. The latter condition was added for mistake (left from old versions). I'll remove it.